### DISCUSSION TITLE
HmIP-STHO-A ist nach Firmware Update auf 1.0.8 ca. 50% der Zeit nicht erreichbar

### DISCUSSION THREAD

**Discussion by soeren487** [OpenCCU/OpenCCU#2628] [🇩🇪 Fragen&Antworten]
Hi Community,

ich habe einen HmIP-STHO-A welchen ich Ende letzten Jahres auf Firmware 1.0.8 geupdated habe. Ich nutze eine RaspberryMatic mit einem Raspberry 3B+ (Set von ELV) und neustem Softwarestand 3.73.9.20231130.

Ich habe den Sensor bereits mehrfach auf Werkseinstellung zurückgesetzt, aus der RaspberryMatic gelöscht, neu angelernt und auch die Batterien zur Sicherheit getauscht. Das Verhalten der etwa 50% Nichterreichbarkeit bleibt unverändert. Er ist für längere Zeit nicht erreichbar, dann meldet er sich wieder.

Hat jemand von euch dieses Verhalten ebenfalls bei sich beobachten können?

Vielen Dank und Gruß
Sören

**Comment by heitec11** [comment]
Hallo,
ich hatte auch Probleme, dass immer mehr Geräte mit Gerätekommunikation nicht mehr erreichbar waren.
Unteranderem der HmIP-STHO, welcher zwischendurch mal einen Wert mitten in der Nacht gesendet hat.
Batterien getauscht, angelernt, HAP's getauscht, alles ohne Verbesserung.

Habe zufällig https://homeitems.de/ccu-geraetekommunikation-gestoert-fehlersuche-teil-1/ gefunden und den HomematicManager aus Teil 2 installiert. (Version 2.7.0; die neuere Version 2.7.1 läuft nicht bei mir und auch vielen anderen)

Letztendlich sieht es bei mir so aus, das Rasbperrymatic 3.73.9.20231130 die Ursache war.
Nach einem Downgrade auf 3.71.12.20231020 läuft es bei mir wieder Fehlerfrei.

**Reply by soeren487** [reply]
@heitec11 Vielen Dank für deine ausführliche Antwort! Ich versuche das mal bei Zeiten umzusetzen.

**Comment by M-Schoeler** [comment]
Hallo Zusammen,
das Problem scheint mit der Raspberrymatic 3.77.7.20240826 immer noch zu bestehen. Mein STHO erzeugt oft die Servicemeldung "kommunikation gestört" auch wenn er nur 2m (ohne Wand) vom  RPI-RF-MOD entfernt ist. Irgendwann erholt er sich wieder von alleine.
Nachdem ich nicht genau weiß, wie ich die Funkverbindung debugge habe ich aus verzweifelung mal die Firmware 1.0.6 auf den Sensor installiert - dann funktioniert er stabiler aber bei weitem nicht perfekt. Anstatt ~50% ausfallzeit sind es vielleicht noch 20%.
Kann mir jemand auf die schnelle verraten, wie ich das genauer debuggen kann?
Anscheinend scheint da ja ein gelegentlich auftretendes Problem mit dem STHO Sensor zu sein?!
(https://homematic-forum.de/forum/viewtopic.php?f=69&t=81561&p=807043&hilit=STHO#p807043)

Gruß Micha
